source/DomainTheory/index.lagda

```diff
@@ -58,18 +58,20 @@ import DomainTheory.Basics.WayBelow        -- (7)
    split support (j.w.w. Ayberk Tosun; added 4 & 5 October 2023)
 3. The theory of continuous/algebraic dcpos
 4. A discussion on possible definitions of continuous dcpo
-5. The Ind-completion is used to discuss the notion of (structurally/pseudo-)
+5. Continuous/algebraic dcpos in an impredicative setting
```

Comment:
@9Y0 Could you please add something like "(Added by Simcha van Collem on ...)" to clarify that this is not part of the original development? Thanks!